std/string.d

```diff
@@ -3641,25 +3671,13 @@ in
 }
 body
 {
-    import std.exception : assumeUnique;
-
-    char[] transTable = new char[256];
+    immutable(char)[256] result = void;
 
-    foreach (i; 0 .. transTable.length)
-        transTable[i] = cast(char)i;
-    foreach (i; 0 .. from.length)
-        transTable[from[i]] = to[i];
-
-    return assumeUnique(transTable);
-}
-
-///
-@safe pure nothrow unittest
-{
-    auto transTable1 = makeTrans("eo5", "57q");
-    assert(translate("hello world", transTable1) == "h5ll7 w7rld");
-
-    assert(translate("hello world", transTable1, "low") == "h5 rd");
+    foreach (i; 0 .. result.length)
+        cast(char)(result[i]) = cast(char)i;
```

Comment:
Yeah, considering that `result` is declared as `immutable(char)[256]`, this is a really bad thing to do.